director/director.go

```diff
@@ -1321,6 +1323,32 @@ func collectClientVersionMetric(reqVer *version.Version, service string) {
 	metrics.PelicanDirectorClientVersionTotal.With(prometheus.Labels{"version": shortendVersion, "service": service}).Inc()
 }
 
+func collectDirectorRedirectionMetric(ctx *gin.Context, destination string) {
+	labels := prometheus.Labels{
+		"destination": destination,
+		"status_code": strconv.Itoa(ctx.Writer.Status()),
+		"version":     "",
+		"network":     "",
+	}
+
+	version, _, err := extractVersionAndService(ctx)
+	if err != nil {
+		log.Warningf("Failed to extract version and service from request: %v", err)
+		return
+	}
+	if version == nil {
+		labels["version"] = "unknown"
+		return
+	}
+	labels["version"] = version.String()
+
+	maskedIp, ok := utils.ApplyIPMask(ctx.ClientIP())
+	if ok {
+		labels["network"] = maskedIp
+	}
```

Comment:
Should we set this to "unknown" if the IP mask comes back as "ok"? An "unknown" in the metric will look better than an empty string because it'll look less like we didn't handle something.